code/game/objects/items/toys.dm

```diff
@@ -925,6 +925,13 @@
 	icon_state = "kittenplushie"
 	slot_flags = SLOT_HEAD
 
+/obj/item/toy/plushie/pennyplush
+    name = "Penny plush"
+    desc = "It's a plush of the beloved company mascot cat, Penny! For the price Nanotrasen sells these things at, you probably could have bought an actual cat."
+    icon_state = "pennyplushie"
+    slot_flags = SLOT_HEAD
```

Comment:
The indentation here (as in, the spaces before the variable names) needs to be tabs and not spaces.

Reply:
Oof. Reuploaded toys.dm with a fix.